### DISCUSSION TITLE
Add link to my checkbox

### DISCUSSION THREAD

**Discussion by erakastrati** [Budibase/budibase#2314] [Help]
I have something like this in my mind. 
I have a table A and it contains columns and rows. I just added a checkbox in there, and I have the idea that when the button is checked, I want to add that information on another table, but it seems like there is no way to add a link to the checkbox.
Does anyone have any idea about this particular problem?
Thanks,
Era.

**Comment by joebudi** [comment]
Hey!

Could you provide more information? 

Is the following a similar scenario: 
You have a to-do table with multiple to-dos, and when a to-do is clicked, it's marked as done, and moved to the done table

**Reply by erakastrati** [reply]
Hi Joe, 
I have an idea like this. I have a table named Project Breakdown, which contains rows and has a clickable checkbox. At the moment that I check that checkbox, I want that the row that I have checked(first picture) to be added on "Shitja dhe marketingu" table. (second picture)
Is that possible?
<img width="1396" alt="Screen Shot 2021-08-09 at 4 37 29 PM" src="https://user-images.githubusercontent.com/74623805/128724447-fc88b6ee-7082-4cec-a3d3-4c06245744a5.png">
<img width="1378" alt="Screen Shot 2021-08-09 at 4 40 11 PM" src="https://user-images.githubusercontent.com/74623805/128724819-c69bf344-5e33-4cea-9aa6-9fbcb210679f.png">

**Reply by joebudi** [reply]
In the data section of Budibase, are these both (project breakdown and Shitja dhe marketing) stored in separate data tables? Or are they stored in the same table?

**Reply by erakastrati** [reply]
They are found on different tables. Can we make a relation?

**Reply by joebudi** [reply]
Out of curiosity, why did you not store both in the same table, with an additional options column; marketing/project to determine what category the row belonged to?

**Reply by erakastrati** [reply]
I don't really know, that is the way that they gave the work to us. I have just to construct it.

**Reply by joebudi** [reply]
Ahhh right. The reason I asked, is if the two tables have the same columns, then they could be the same table with a single column that differs the rows. If that was the case, the solution is pretty trivial. If not, we may need to use an automation.

**Reply by erakastrati** [reply]
Thanks a lot Joe. It takes serious courage to get on this path and stay on it. 
Good on you 🙂
